### DISCUSSION TITLE
No volume curve present in the timeline

### DISCUSSION THREAD

**Discussion by andrei-zgirvaci** [remotion-dev/remotion#690] [Q&A]
I am not sure why, but I don't see any **volume curve draw in the timeline** as described in the documentation:

*You may also pass a callback function that returns the volume based an arbitrary frame number. This has the benefit that Remotion is able to draw a volume curve in the timeline!*

Link: https://www.remotion.dev/docs/using-audio#controlling-volume

I made sure to write my interpolate function accordingly:

```js
<Audio
  src={episodeAudio}
  startFrom={startFrom - 1}
  endAt={endAt}
  volume={(f) =>
    interpolate(
      f,
      [START_AUDIO_FADE_OUT_FROM, videoConfig.durationInFrames],
      [1, 0],
      { extrapolateLeft: "clamp" }
    )
  }
/>;
```

This is what I see in the timeline:
![Screenshot 2021-11-07 at 2 21 42 PM](https://user-images.githubusercontent.com/7466068/140634836-f457eb96-c29f-4f1d-a6ad-7fd268617ef1.JPG)

I guess I don't even know what I am expected to see, maybe it's already present, but I just don't know what to look for.

I would appreciate it if someone will show an example of their timeline with the volume curve drawn :)

**Comment by JonnyBurger** [comment]
It seems a line is there, but it's at the top of the layer, but it's always at full volume and therefore it's sticking at the top of the layer.

Note that if you use a callback function, the frame of where the audio starts playing is always `0`, irrespective of what frame is shown in the timeline. So instead of using `videoConfig.durationInFrames`, maybe it's better to use `endAt` - `startAt`.

You can debug it really easily by just using `console.log()` for your volume and seeing if you are always at volume 1 😃
